src/ui/screens/Parachains/ParachainDetailScreen.tsx

```diff
@@ -1,102 +1,38 @@
-import type {
-  AccountId,
-  CoreAssignment,
-  ParaValidatorIndex,
-  ParaId,
-  CandidatePendingAvailability,
-} from '@polkadot/types/interfaces';
 import {EmptyView} from '@ui/components/EmptyView';
-import IdentityIcon from '@polkadot/reactnative-identicon/Identicon';
-import {useAccountIdentityInfo} from 'src/api/hooks/useAccountIdentityInfo';
-import AccountInfoInlineTeaser from '@ui/components/AccountInfoInlineTeaser';
-import {useParachainInfo} from 'src/api/hooks/useParachainInfo';
-import {useParaEndpoints} from 'src/api/hooks/useParaEndpoints';
-import {notEmpty} from 'src/utils';
+import {AccountInfo} from 'src/api/hooks/useParachainsOverview';
+import {AccountTeaser} from '@ui/components/Account/AccountTeaser';
 
 type ScreenProps = {
   route: RouteProp<ParachainsStackParamList, 'Parachain'>;
 };
 
-type ParachainValidators = {
-  validators: AccountId[];
-  assignments?: CoreAssignment[];
-  validatorGroups: ParaValidatorIndex[][];
-  validatorIndices?: ParaValidatorIndex[];
-};
-
-function getValidatorInfo(id: string, parachainValidators?: ParachainValidators) {
-  const assignment = parachainValidators?.assignments?.find(({paraId}) => paraId.eq(id));
-
-  if (!assignment) {
-    return undefined;
-  }
-
-  return {
-    groupIndex: assignment.groupIdx,
-    validators: parachainValidators?.validatorGroups[assignment.groupIdx.toNumber()]
-      ?.map((indexActive) => [indexActive, parachainValidators?.validatorIndices?.[indexActive.toNumber()]])
-      .filter(([, a]) => a)
-      .map(([, indexValidator]) =>
-        indexValidator ? parachainValidators.validators[indexValidator?.toNumber()] : undefined,
-      )
-      .filter(notEmpty),
-  };
-}
-
-function getNonVoters(validators?: AccountId[], pendingAvail?: CandidatePendingAvailability) {
-  let list: AccountId[] = [];
-
-  if (validators && pendingAvail) {
-    list = pendingAvail.availabilityVotes
-      .toHuman()
-      .slice(2)
-      .replace(/_/g, '')
-      .split('')
-      .map((c, index) => (c === '0' ? validators[index] : null))
-      .filter((v, index): v is AccountId => !!v && index < validators.length);
-  }
-
-  return list;
-}
-
 export function ParachainDetailScreen({route}: ScreenProps) {
-  const {id, name, period, blocks} = route.params;
-  const events = useParachainEvents();
-  const {data: parachainValidators} = useParachainValidators();
-  const {data: parachainInfo} = useParachainInfo(id as unknown as ParaId);
-
-  const validatorInfo = useMemo(() => getValidatorInfo(id, parachainValidators), [id, parachainValidators]);
-  const nonVoters = useMemo(
-    () => getNonVoters(parachainValidators?.validators, parachainInfo?.pendingAvail),
-    [parachainValidators, parachainInfo],
-  );
-
-  const endpoints = useParaEndpoints(id as unknown as ParaId);
-  const homepage = endpoints?.length ? endpoints[0]?.homepage : undefined;
+  const {id, name, lastIncludedBlock, lastBackedBlock, lease, lifecycle, homepage, nonVoters, validators} =
```

Comment:
This data should come from another hook: `useParachain(route.params.paraId)`. We don't want to over fetch data in the overview screen.